drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/commands/CommandExecutor.java

```diff
@@ -187,7 +197,21 @@ public void handle(ScenarioGridReloadEvent event) {
 
     @Override
     public void onEvent(SetColumnValueEvent event) {
-        commonExecute(new SetColumnValueCommand(model, String.valueOf(new Date().getTime()), event.getFullPackage(), event.getValue(), event.getValueClassName(), scenarioGridPanel, scenarioGridLayer));
+        if (model.getSelectedColumn() == null) {
+            return;
+        }
+        if (model.isSelectedColumnEmpty()) {
+            commonExecute(new SetColumnValueCommand(model, String.valueOf(new Date().getTime()), event.getFullPackage(), event.getValue(), event.getValueClassName(), scenarioGridPanel, scenarioGridLayer, false));
+        } else if (model.isSameSelectedColumnProperty(event.getValue())) {
+            return;
+        } else if (model.isSameSelectedColumnType(event.getValueClassName())) {
+            Command yesCommand = () -> commonExecute(new SetColumnValueCommand(model, String.valueOf(new Date().getTime()), event.getFullPackage(), event.getValue(), event.getValueClassName(), scenarioGridPanel, scenarioGridLayer, false));
+            Command noCommand = () -> commonExecute(new SetColumnValueCommand(model, String.valueOf(new Date().getTime()), event.getFullPackage(), event.getValue(), event.getValueClassName(), scenarioGridPanel, scenarioGridLayer, true));
+            yesNoConfirmPopupPresenter.show(ScenarioSimulationEditorConstants.INSTANCE.updateColumn(), ScenarioSimulationEditorConstants.INSTANCE.dataInColumnKeepOrDelete(), InlineNotification.InlineNotificationType.WARNING, CommonConstants.INSTANCE.Delete(), ScenarioSimulationEditorConstants.INSTANCE.keep(), Button.ButtonStyleType.DANGER, Button.ButtonStyleType.DANGER, null, yesCommand, noCommand);
```

Comment:
@gitgabrio I see this exceeding of recommended amount of chars per line in scenario-simulation often and I am not asking for fix as it is ont official requirement for code style but in this case I will make an exception, could you plase format as bellow? I know it is caused by inapproparite `show` method signature but will really facilitate reading and reviewing code.
```java
yesNoConfirmPopupPresenter.show(ScenarioSimulationEditorConstants.INSTANCE.updateColumn(),
                                            ScenarioSimulationEditorConstants.INSTANCE.dataInColumnKeepOrDelete(),
                                            InlineNotification.InlineNotificationType.WARNING,
                                            CommonConstants.INSTANCE.Delete(),
                                            ScenarioSimulationEditorConstants.INSTANCE.keep(),
                                            Button.ButtonStyleType.DANGER,
                                            Button.ButtonStyleType.DANGER, 
                                            null, 
                                            yesCommand,
                                            noCommand);
```